src/js/components/Form/index.d.ts

```diff
@@ -1,11 +1,16 @@
 import * as React from "react";
 
+export interface FormExtendedEvent extends React.FormEvent {
```

Comment:
does it make sense that this take a type param?

i.e.,

```ts
export interface FormExtendedEvent<T = Element> extends React.FormEvent<T> {
  value: Record<string, any>;
  touched: Record<string, boolean>;
}
```

since the react type is

```ts
interface FormEvent<T = Element> extends SyntheticEvent<T> {
}
```

Reply:
Actually, you are right, wasn't attentive